src/main/java/org/codehaus/mojo/taglist/FileAnalyser.java

```diff
@@ -73,7 +72,7 @@ public class FileAnalyser {
     /**
      * The Locale of the files to analyze.
      */
-    private final Locale locale;
+    private final Locale sourceFileLocale;
```

Comment:
`sourceFilesLocale`, no?

Reply:
We have a `sourceFileLocale` parameter of Mojo report ... so let to be consistent

Reply:
Yes, both need to be consistent with a TODO.